packages/pds/src/handle/moderation/index.ts

```diff
@@ -0,0 +1,66 @@
+import * as ui8 from 'uint8arrays'
+import AppContext from '../../context'
+import { REASONOTHER } from '../../lexicon/types/com/atproto/moderation/defs'
+import { UnacceptableHandleValidator } from './validator'
+
+// regexes taken from: https://github.com/Blank-Cheque/Slurs
+/* eslint-disable no-misleading-character-class */
+const explicitSlurRegexes = [
```

Comment:
There is no license in the repo you stole this code from, meaning this code is copyrighted and you do not have permission to use it. I would expect devs working on an open source project not to do something as basic as literally ripping code from copyrighted material but oh well.

Reply:
> I'm no lawyer, but I'd kindly ask that you leave the comment included for credit.
This is a license. The implied agreement is to use it, it'd be nice if you left a comment.

Reply:
> This is a license. The implied agreement is to use it, it'd be nice if you left a comment.

That is not a license. I'd encourage you to read some of the licenses here: https://opensource.org/licenses/

Furthermore *if* that was even a license, it would only be for use for Reddit Automod use, as is made clear in the README.

Reply:
Not every license has to be in a license file. It's not a formal license sure, but a license is just a way of handing over rights when you are initially given none. The readme which is the only place rights are specified does not say it is for Reddit only, and only says that "but I'd kindly ask that you leave the comment included for credit". This implies you can use it due to the lack of restrictions and that one condition being specified in what is effectively a block of the readme that is a pseudo-license.

Reply:
> > This is a license. The implied agreement is to use it, it'd be nice if you left a comment.
> 
> That is not a license. I'd encourage you to read some of the licenses here: https://opensource.org/licenses/
> 
> Furthermore _if_ that was even a license, it would only be for use for Reddit Automod use, as is made clear in the README.

The license is clearly stated in the README being that you can use it if you give credit. Licenses don't have to be in a specific file nor do they have to be super formal. You're just unnecessarily trying to cause issues.